plugins/cloudservices/plugin.js

```diff
@@ -75,22 +75,55 @@
 		},
 
 		beforeInit: function( editor ) {
+			var tokenUrl = editor.config.cloudServices_tokenUrl,
+				tokenFetcher = {
+					token: null,
+
+					// Allow external code (tests) to affect token refresh interval if needed.
+					REFRESH_INTERVAL: editor.CLOUD_SERVICES_TOKEN_INTERVAL || 3600000,
+
+					refreshToken: function() {
+						CKEDITOR.ajax.load( tokenUrl, function( token ) {
+							if ( token ) {
+								tokenFetcher.token = token;
+							}
+						} );
+					},
+
+					init: function() {
```

Comment:
This method could be simplified:

* first call to `refreshToken` could use `this`
* `setInterval` could just take `this.refreshToken` and `this.REFRESH_INTERVAL`. There is no need to introduce anonymous function or `that` variable.

Reply:
Good point.